packages/client/package.json

```diff
@@ -5,7 +5,8 @@
 	"author": "Alistair Smith <hi@alistair.sh>",
 	"description": "Functional HTTP Framework for TypeScript",
 	"scripts": {
-		"build": "tsup"
+		"build": "tsup",
+		"attw": "npx --yes @arethetypeswrong/cli --profile node16 --pack ."
```

Comment:
Could we get `@arethetypeswrong/cli` installed in the root package JSON and then avoid using `npx` here?

"attw": "@arethetypeswrong/cli --profile node16 --pack ."